plugin/cli.go

```diff
@@ -0,0 +1,142 @@
+package plugin
+
+import (
+	"errors"
+	"fmt"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+const (
+	filenameFlag = "filename"
+	pathFlag     = "path"
+	mountFlag    = "mount"
+	rebuildFlag  = "rebuild"
+	restoreFlag  = "restore"
+
+	debugFlag = "debug"
+
+	repoOwnerFlag    = "repo.owner"
+	repoNameFlag     = "repo.name"
+	commitBranchFlag = "commit.branch"
+)
```

Comment:
Using constants here would be non-standard for most Go programmers. I can see why you are doing it, but you won't see this pattern used much with `urfav/cli`